diff -uraN droidcam-obs-plugin-2.4.2/linux/gui.mk omv-droidcam-obs-plugin-2.4.2/linux/gui.mk
--- droidcam-obs-plugin-2.4.2/linux/gui.mk	2025-10-13 07:52:42.000000000 +0200
+++ omv-droidcam-obs-plugin-2.4.2/linux/gui.mk	2025-11-07 02:14:37.309570032 +0100
@@ -1,6 +1,8 @@
 # Copyright (C) 2022 DEV47APPS, github.com/dev47apps
 
 ifdef DROIDCAM_OVERRIDE
+MOC      ?= /usr/lib64/qt6/libexec/moc
+UIC      ?= /usr/lib64/qt6/libexec/uic
 INCLUDES += -Isrc/ui
 SRC      += src/ui/AddDevice.cpp
 SRC      += src/ui/moc_AddDevice.cpp
@@ -17,12 +19,7 @@
 	sudo cp $(LIB_DLL) /opt/droidcam-obs-client/obs-plugins/64bit/
 	sudo cp -R data/*  /opt/droidcam-obs-client/data/obs-plugins/droidcam-obs/
 
-#todo - upgrade
-$(eval $(call ADD_Lib,Qt5Gui))
-$(eval $(call ADD_Lib,Qt5Svg))
-$(eval $(call ADD_Lib,Qt5Widgets))
-else
-
+$(eval $(call ADD_Lib,Qt6Core))
 $(eval $(call ADD_Lib,Qt6Gui))
 $(eval $(call ADD_Lib,Qt6Svg))
 $(eval $(call ADD_Lib,Qt6SvgWidgets))
